applications/OptimizationApplication/python_scripts/analyses_controller.py

```diff
@@ -27,34 +27,34 @@ def __init__(self, execution_policies_settings, model, model_parts_controller):
 
         self.execution_policy_wrappers = {}
```

Comment:
```suggestion
        self.execution_policy_decorators = {}
```

Rename wrapper -> decorator

Reply:
done :)